packages/app/src/elements/org-settings.ts

```diff
@@ -169,4 +210,55 @@ export class OrgSettingsView extends Routing(StateMixin(LitElement)) {
             </div>
         `;
     }
+
+    private _renderDirectorySettings() {
+        const org = this._org!;
+
+        let sectionHtml: TemplateResult<1>;
+
+        if (org.directory.syncProvider !== "none") {
+            const scimSecret = bytesToBase64(org.directory.scim!.secret, true);
+            // TODO: Make this section more helpful and pretty
+            // TODO: Get proper SCIM host + port
```

Comment:
@MaKleSoft I haven't figured out how to get the server's URL and port for the frontend by "forcefully inspecting" :sweat_smile: let me know if there's a way or if I need to come up with something.

Reply:
I think this should be provided as `org.directory.scim.url` or something, that is set on the server side, probably [here](https://github.com/padloc/padloc/blob/feature/scim-v1/packages/core/src/server.ts#L1037).